cfme/services/catalogs/orchestration_template.py

```diff
@@ -68,18 +70,23 @@ def __init__(self, template_type=None, template_name=None, description=None):
         self.template_name = template_name
         self.description = description
 
-    def create(self, content):
+    def create_form(self, content):
         sel.force_navigate('create_new_template',
                            context={'template_type': self.template_type})
         if(self.template_type == "CloudFormation Templates"):
             temp_type = "Amazon CloudFormation"
         else:
             temp_type = "OpenStack Heat"
-        fill(create_template_form, {'template_name': self.template_name,
-                                    'description': self.description,
+        fill(create_template_form, {'description': self.description,
                                     'template_type': temp_type,
-                                    'content': content},
+                                    'content': content,
+                                    'template_name': self.template_name},
             action=create_template_form.add_button)
+
+    def create(self, content):
+        self.create_form(content)
+        if(flash.message_contain("Md5 of content already exists")):
```

Comment:
Should we, instead of this, as part of the test, delete all the stack templates before starting the test as a fixture?